src/quo/components/share/share_qr_code/view.cljs

```diff
@@ -179,11 +186,13 @@
      - emoji:               Emoji in a string to show in the QR code.
      - on-legacy-press:     Callback for the legacy tab.
      - on-multichain-press: Callback for the multichain tab.
+     - watched-account?     Boolean
    `:wallet-multichain`
      - networks:            A vector of network names as keywords (`[:ethereum, :my-net, ...]`).
      - on-settings-press:   Callback for the settings button.
      - emoji:               Emoji in a string to show in the QR code.
      - on-legacy-press:     Callback for the legacy tab.
+     - watched-account?     Boolean
```

Comment:
These days I had the pleasure to open a component with malli schemas. Of course, it's not a requirement to write or refactor components to use schemas, but seeing the props change here I couldn't help but comment :)

Reply:
I believe this is the api. 
```
share-qr-type qr-image-uri component-width customization-color full-name
           profile-picture emoji on-share-press]
           profile-picture emoji on-share-press address
```
Although I agree with @ilmotta - it would be better to remove doc string and add a malli spec instead 👍